libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/internal/navigation/FreeDriveLocationUpdater.kt

```diff
@@ -52,18 +61,28 @@ internal class FreeDriveLocationUpdater(
                 }
             }, 1500, 1000, TimeUnit.MILLISECONDS)
         }
+
+        if (electronicHorizonFuture == null) {
+            electronicHorizonFuture = executorService.scheduleAtFixedRate({
+                val request = electronicHorizonRequestBuilder.build(ElectronicHorizonRequestBuilder.Expansion._1D, cachedLocations)
+                mapboxNavigator.retrieveElectronicHorizon(request)
+            }, ELECTRONIC_HORIZON_DELAY, ELECTRONIC_HORIZON_INTERVAL, TimeUnit.MILLISECONDS)
```

Comment:
Do we want to wait 20 seconds for the initial request `ELECTRONIC_HORIZON_DELAY` (`initialDelay`)?

Reply:
now it's configurable. What value is more suitable here?

Reply:
Let's leave it as is and revisit after testing in the field 👍